strawberry/asgi/__init__.py

```diff
@@ -194,12 +194,19 @@ async def handle_async_results(
                 {"data": None, "errors": [format_graphql_error(error)]},
                 operation_id,
             )
+        except asyncio.CancelledError:
+            # CancelError will be received when the client disconnects and the handle_async_results task being cancelled
+            # Suppress the error and prevent contining to the sending completion message stage
+            return
```

Comment:
Can you make sure you've [installed the `pre-commit` hooks](https://github.com/strawberry-graphql/strawberry#pre-commit)? There's some minor formatting issues here, and the imports haven't been sorted.

Reply:
Sorry, I probably forgot that. I will first settle the PR on `graphql-core` then get back here.

Reply:
@wuyuanyi135 does that mean we need to wait for the updated release of `graphql-core` to merge this PR?

Reply:
@patrick91 The fix has been merged in `graphql-core`. We do need the updated release here.

Reply:
@wuyuanyi135 Strawberry was upgraded to graphql-core 3.1.5 (24080d57817dabd1e571f5463a82e4eeaf0b2dde) which includes your upstream changes.